packages/compass-aggregations/src/components/pipeline/pipeline.jsx

```diff
@@ -186,9 +192,51 @@ class Pipeline extends PureComponent {
           restorePipelineFrom={this.props.restorePipelineFrom}
           deletePipeline={this.props.deletePipeline}
           savedPipelinesListToggle={this.props.savedPipelinesListToggle}
-          savedPipeline={this.props.savedPipeline} />
+          savedPipeline={this.props.savedPipeline}
+        />
+      );
+    }
+  }
+
+  renderPipelineToolbar() {
+    if (process.env.COMPASS_SHOW_NEW_AGGREGATION_TOOLBAR !== 'true') {
```

Comment:
On cloud `process` might be undefined 
```suggestion
    if (global?process?.env?.COMPASS_SHOW_NEW_AGGREGATION_TOOLBAR !== 'true') {
```